tests/libs/estdlib/test_tcp_socket.erl

```diff
@@ -56,13 +58,15 @@ test_shutdown() ->
 
     ok = test_shutdown_of_client_sockets(Port),
 
-    ok = close_listen_socket(ListenSocket).
+    ok = close_listen_socket(ListenSocket),
+
+    id(ok).
```

Comment:
I do not understand this line

Reply:
These are to force a stack trace.  Otherwise, you get a tail call, which can make finding failures hard.